lib/async_hooks.js

```diff
@@ -60,19 +60,11 @@ async_wrap.setupHooks({ init,
                         destroy: emitDestroyN });
 
 // Used to fatally abort the process if a callback throws.
-function fatalError(e) {
```

Comment:
What about exporting `FatalException` then? Looking at `FatalException` it seems that there is a lot of complexity that isn't handled by the current approach.